scripts/commit_classifier.py

```diff
@@ -294,7 +294,7 @@ def apply_phab(self, hg, phabricator_deployment, diff_id):
             # Stop as soon as a base revision is available
             if self.has_revision(hg, patch.base_revision):
                 logger.info(
-                    f"Stopping at diff {patch.id} and revision {patch.base_revision}"
+                    "Stopping at diff %f and revision %f", patch.id, patch.base_revision
```

Comment:
Could you clarify why `%f` is used for `patch.id`?

Reply:
It should not be %f.
I am confused between %d and %s, so now i am using .format method